### PR TITLE
Add KNC token

### PR DESCRIPTION
**Pull request by manhlx3006**
**Description**
Added KNC addresses

### CODE REVIEW

**Comment by smartcontracts** [comment]
Are your tokens deployed to Optimism, Optimism Kovan, or Optimism Goerli? We currently do not accept PRs for only L1 token addresses.

File: data/KNC/data.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "symbol": "KNC",
+  "decimals": 18,
+  "tokens": {
+    "ethereum": {
+      "address": "0xdeFA4e8a7bcBA345F687a2f1456F5Edd9CE97202"
+    },
+    "optimism": {
+      "address": "0xa00E3A3511aAC35cA78530c85007AFCd31753819"
+    },
+    "kovan": {
```
Comment (roninjin10) [inline data/KNC/data.json:12]: You will need to include an OptimismKovan address or remove this one for now

**Review by roninjin10 — CHANGES_REQUESTED** [review]
Need to remove kovan address

File: data/KNC/data.json (hunk 1 above)
Comment (manhlx3006) [reply data/KNC/data.json]: PR updated, removed KNC kovan address

**Review by roninjin10 — APPROVED** [review]

**Pull request merged**